imports/plugins/core/dashboard/client/templates/shop/settings/settings.html

```diff
@@ -102,6 +102,7 @@
             {{> afQuickField name='settings.openexchangerates.refreshPeriod' placeholder="every 1 hour"}}
             {{> afQuickField name='settings.google.clientId'}}
             {{> afQuickField name='settings.google.apiKey'}}
+            {{> afFormGroup name='settings.cart.cleanup' options=options placeholder="Every 3 days"}}
```

Comment:
This should just be a text field. Also the name should be more descriptive like `cleanupDurationDays`